### PR TITLE
3.2

### PR DESCRIPTION
**Pull request by olu144**
Modified index.html and script.js to complete part 2

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -38,3 +38,9 @@ function addRandomStock() {
   const stockContainer = document.getElementById('stock-container');
   stockContainer.innerText = stock;
 }
+
+function getGreetingUsingArrowFunctions() {
+  fetch('/data').then(response => response.text()).then((greeting) => {
+    document.getElementById('greeting-container').innerText = greeting;
+  });
+}
```
Comment (dborkan) [inline portfolio/src/main/webapp/script.js:46]: Tiny nit: if you add a new line at the very end of this file (just after the `}`) that little red icon will go away

File: walkthroughs/week-3-server/examples/random-quotes/src/main/webapp/pom.xml [hunk 2]
```diff
@@ -0,0 +1,55 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
```
Comment (dborkan) [inline walkthroughs/week-3-server/examples/random-quotes/src/main/webapp/pom.xml]: Same comment as in PR #2  -  I think this file can be deleted

**Review by dborkan — APPROVED** [review]

**Review by dslucas — APPROVED** [review]

File: walkthroughs/week-3-server/examples/random-quotes/src/main/webapp/pom.xml (hunk 2 above)
Comment (olu144) [reply walkthroughs/week-3-server/examples/random-quotes/src/main/webapp/pom.xml]: Addressed in 3.5part1

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (olu144) [reply portfolio/src/main/webapp/script.js:46]: Addressed in 3.5part1

**Pull request merged**